docs/SPEC.md

```diff
@@ -1197,6 +1194,8 @@ Intervals has the following parameters:
 The Nth interval start date is the initial start date plus the offset plus an Nth multiple of the every parameter.
 Each interval stop date is equal to the interval start date plus the period duration.
 When filtering intervals each potential interval is passed to the filter function, when the function returns false, that interval is excluded from the set of intervals.
+The duration for an interval must contain duration units for only one base unit as defined in the duration literals.
+It is not possible to mix months and nanoseconds in an interval.
```

Comment:
I think we should be more specific about the `every` parameter, otherwise it will be confusing which parameters allow mixed and which do not.